### PR TITLE
Feature/tex 4957 cert mon example

### PR DESCRIPTION
**Pull request by JohnEllwoodBJSS**

### CODE REVIEW

File: tools/acm-cert-monitor/resources/acm_cert_monitor.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+def send_msg_slack(result):
+
+   url= get_slack_url()
+   if(url is None):
+      print ('unable to send slack message, slack url not set')
+   else:     
+      msg = {
+        "channel": "#texas-alerts",
```
Comment (andyblundell) [inline tools/acm-cert-monitor/resources/acm_cert_monitor.py:95]: Can we anonymise that reference to Texas please?

**Review by andyblundell — CHANGES_REQUESTED** [review]
Looks great, thanks - just noticed one reference we might want to remove

File: tools/acm-cert-monitor/resources/acm_cert_monitor.py (hunk 1 above)
Comment (JohnEllwoodBJSS) [reply tools/acm-cert-monitor/resources/acm_cert_monitor.py:95]: Good spot!

**Review by andyblundell — APPROVED** [review]
Looks great, thanks

**Pull request merged**